package.json

```diff
@@ -552,6 +562,11 @@
           "when": "vscode-objectscript.connectActive && resourceScheme == file",
           "group": "objectscript_modify@1"
         },
+        {
```

Comment:
As commented re the editor/context menu, perhaps don't add this to explorer/context.